UnoCheck/Checkups/WindowsPhytonInstallationCheckup.cs

```diff
@@ -17,25 +16,35 @@ public class WindowsPhytonInstallationCheckup : Checkup
 
 		public override async Task<DiagnosticResult> Examine(SharedState history)
 		{
-			RegistryKey pyLaucherKey = Registry.LocalMachine.OpenSubKey(@"SOFTWARE\WOW6432Node\Python\PyLauncher", false);
-			string? path = pyLaucherKey?.GetValue(InstallDirKey)?.ToString();
-
-			if (pyLaucherKey is null || string.IsNullOrEmpty(path))
+			if (!this.PhytonIsPresent())
```

Comment:
This looks unrelated ?

Reply:
I found an issue when I working on GTK3. I just changed the way to detect Python.